sdks/java/core/src/main/java/org/apache/beam/sdk/values/Row.java

```diff
@@ -381,7 +381,7 @@ public static Builder withSchema(Schema schema) {
       this.schema = schema;
     }
 
-    public Builder addValue(Object values) {
+    public Builder addValue(@Nullable Object values) {
```

Comment:
I added annotation because findbugs complained, it's already widely used with nullable values, just wasn't caught before, meaning there would be only one usage of `addNullValue`